src/main/java/com/ebivariation/contigalias/controller/ContigAliasController.java

```diff
@@ -76,14 +81,16 @@ public ContigAliasController(AssemblyService assemblyService, ChromosomeService
         }
     }
 
-    @GetMapping(value = "chromosomes/genbank/{genbank}")
+    @ApiOperation(value = "Get an chromosome using it's Genbank accession.")
+    @GetMapping(value = "chromosomes/genbank/{genbank}", produces = "application/json")
     public ResponseEntity<ChromosomeEntity> getChromosomeByGenbank(@PathVariable String genbank) {
         Optional<ChromosomeEntity> entity = chromosomeService.getChromosomeByGenbank(genbank);
         return entity.map(chromosomeEntity -> new ResponseEntity<>(chromosomeEntity, HttpStatus.OK))
                      .orElseGet(() -> new ResponseEntity<>(HttpStatus.NOT_FOUND));
     }
 
-    @GetMapping(value = "chromosomes/refseq/{refseq}")
+    @ApiOperation(value = "Get an chromosome using it's Refseq accession.")
```

Comment:
"its" here is a possessive. Change here and the other similar sentences.
```suggestion
    @ApiOperation(value = "Get an chromosome using its Refseq accession.")
```